src/components/Navbar.jsx

```diff
@@ -10,17 +10,13 @@ export default function NavBar() {
       <NavLink
       exact
       to="/"
-      className="px-4 py-2 bg-base-100 text-primary rounded-full font-bold hover:bg-red-500 hover:text-white"
-      activeClassName="bg-red-500 text-white border-red-500"
-      >
+      className="mx-4 bg-base-100 px-3 text-primary hover:border-b-4 border-b-[#0A2E50] font-bold text-[2.1vh] ">
         About Us
       </NavLink>
       <NavLink
       to="/professionals"
-      className="px-4 py-2 bg-base-100 text-primary rounded-full font-bold hover:bg-red-500 hover:text-white"
-      activeClassName="bg-red-500 text-white border-red-500"
-      >
-        Resources
+      className="px-4 py-[8px] rounded-full font-bold bg-red-500 text-white text-[2.1vh] hover:bg-gradient-to-r from-red-500 to-purple-500 ">
+        Unmatched Voices
       </NavLink>
     </div>
```

Comment:
```
<div className="space-x-6 md:space-x-8 text-sm md:text-base xl:text-lg">
      <NavLink
        exact
        to="/"
        className="text-primary font-bold flex-col justify-start items-center inline-flex relative group"
      >
        <span className="text-center font-bold md:group-hover:opacity-100">
          About Us
        </span>
        <span className="w-full md:h-1 h-[3px] top-5 md:top-full bg-primary rounded-2xl absolute bottom-0 md:opacity-0 transition-opacity duration-300 ease-in-out opacity-100 md:group-hover:opacity-100" />
      </NavLink>
      <NavLink
        exact
        to="/"
        className="text-primary font-bold flex-col justify-start items-center inline-flex relative"
      >
        <span className="md:px-4 md:py-1 rounded-full md:bg-red-500 md:text-white md:hover:bg-gradient-to-r from-red-500 to-purple-500 text-center font-bold">
          Unmatched Voices
        </span>
        <span className="w-full md:h-1 h-[3px] top-5 md:top-full bg-primary rounded-2xl absolute bottom-0 md:opacity-0 transition-opacity duration-300 ease-in-out opacity-100 md:group-hover:opacity-100" />
      </NavLink>
    </div>
```

Reply:
instead of using `border-b` im using a span to act as the underline so that its easier to style.

Reply:
Thank you so much for this Sherry! I totally forgot about the mobile view being different. its updated now but the active class for mobile screen isn't working for now

Reply:
Everything is rectified now!